File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Actions/Operation/ApplicationLogin.cs
using Cen.Wms.Client.Actions.Data.Auth;
using Cen.Wms.Client.Actions.UI.Auth;
using Cen.Wms.Client.Models.State;
using Cen.Wms.Client.Services;

namespace Cen.Wms.Client.Actions.Operation
{
    class ApplicationLogin
    {
        public static bool Run()
        {
            var result = false;

            var credentials = AskForUserCredentials.Run();
            if (credentials != null)
            {
                var userToken = UserLogin.Run(credentials);
                GStateProvider.Instance.SetStateAuth(new StateAuth(userToken));
            }

            return GStateProvider.Instance.StateAuth.IsAuthorised;
        }
    }
}

File: TCDNew-master/src/ui.android/lib/actions/operation/ApplicationLogin.dart
import 'package:tiq/actions/data/auth/UserLogin.dart';
import 'package:tiq/actions/ui/auth/AskForUserCredentials.dart';
import 'package:flutter/material.dart';
import 'package:tiq/models/state/StateAuth.dart';
import 'package:tiq/services/GStateProvider.dart';

class ApplicationLogin {
  Future<bool> run(BuildContext context) async {
    final credentials = await AskForUserCredentials().run(context);
    if (credentials != null) {
      final userToken = await UserLogin().run(context, credentials);
      GStateProvider.instance.setStateAuth(
        StateAuth(userToken)
      );
    }

    return GStateProvider.instance.stateAuth.isAuthorised;
  }
}
